web-bff/Makefile

```diff
@@ -14,9 +15,17 @@ setup:
 run:
 	@poetry run uvicorn app.main:app --host 0.0.0.0 $(if $(port),--port $(port)) --reload
 
+test-report:
+	@poetry run pytest --asyncio-mode=auto --cov=app --cov-report=html:coverage-reports --html="test-reports/test_report.html" --self-contained-html tests/
+
 test:
 	@poetry run pytest --asyncio-mode=auto --cov=app tests/
 
+poetry-add:
```

Comment:
Need to add the documentstion in the `CONTRIBUTING.md` for this new make target.